Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2205,6 +2205,20 @@ DeprecationReportingEnabled:
     WebCore:
       default: false
 
+DetailsNameAttributeEnabled:
+  type: bool
+  status: stable
+  category: html
+  humanReadableName: "HTML <details name> attribute"
+  humanReadableDescription: "Enable HTML <details name> attribute"
+  defaultValue:
+    WebKitLegacy:
+      default: true
+    WebKit:
+      default: true
+    WebCore:
+      default: true
```

Comment:
Upon reflection, can we enable it only in STP ? Mostly to assess the state of the spec before enabling.

```suggestion
DetailsNameAttributeEnabled:
  type: bool
  status: preview
  category: html
  humanReadableName: "HTML <details name> attribute"
  humanReadableDescription: "Enable HTML <details name> attribute"
  defaultValue:
    WebKitLegacy:
      default: false
    WebKit:
      default: false
    WebCore:
      default: false
```